xtask/src/hooks.rs

```diff
@@ -1,12 +1,14 @@
 use std::fs::File;
```

Comment:
Add our Python checks and tests if we opt-in. Since we can't guarantee Python on the host machine, nor the presence of tox or flake8, we don't enable these by default.